src/sugar3/graphics/progressicon.py

```diff
@@ -14,11 +14,19 @@
 # License along with this library; if not, write to the
 # Free Software Foundation, Inc., 59 Temple Place - Suite 330,
 # Boston, MA 02111-1307, USA.
+'''
+The progressicon module provide a progress icon; a widget like progress
+bar which shows progress of a task 
+'''
 
 from gi.repository import Gtk
 from sugar3.graphics.icon import get_surface
 from sugar3.graphics import style
 
+'''
+The above three lines are for importing modules. The first one imports GTK form gi.repository, second one imports get_surface from 
+sugar3.graphics.icon and third one imports style namespace from sugar3.graphics.
+'''
```

Comment:
As stated earlier, there is no need for the above paragraph as it is self-explanatory